common/views/components/ImageViewer/ImageViewer2.js

```diff
@@ -63,6 +64,16 @@ class ViewerContent extends React.Component<ViewerContentProps> {
     document.removeEventListener('keydown', this.escapeCloseViewer);
   }
 
+  handleZoom = (event: {currentTarget: {title: string}}) => {
+    const isZoomIn = event.currentTarget.title === 'Zoom in';
```

Comment:
Hmm. Again - seems like we're relying on DOM elements, instead of state and data?

Reply:
This was an alternative to writing a `handleZoomIn` and `handleZoomOut` method (but I think I maybe would prefer to write both, so this messiness would go away).